src/GUI/UI/Books/JournalBook.cpp

```diff
@@ -23,18 +25,18 @@
 
 Image *ui_book_journal_background = nullptr;
 
+static int currentIdx;
+static std::vector<int> journalIdx;
+static std::vector<int> journalEntryPage;
```

Comment:
Same Q here – can these statics go as fields somewhere?

If not, we'll need a TODO.